modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/pages/v2/additional_issues.rb

```diff
@@ -16,6 +16,14 @@ def build!
 
               pdf.start_new_page
 
+              unless short_veteran_email?
+                pdf.text "\n<b>Veteran Email:</b>\n#{form_data.veteran_email}\n", inline_format: true
```

Comment:
Should this be on it's own page? Or should it be with the additional issues as you have it?

Reply:
I thought about that too, but I figured it'd be easier on the intake specialist if we just let additional info on attached pages page break wherever they needed to, rather than risking an email page (which is otherwise mostly blank) and issues page(s).